README.md

```diff
@@ -20,6 +22,10 @@ This is the early stages of a UDS Bundle we call the UDS Software Factory. This
 
 ## Prerequisites
 
+### UDS-CLI
+
+- Install the release of [uds-cli](https://github.com/defenseunicorns/uds-cli/releases) that matches the version in the [makefile](https://github.com/defenseunicorns/uds-package-software-factory/blob/main/Makefile#L4).
```

Comment:
Should we reference the make target that will do this for you automatically?

Reply:
This is being written under the assumption that the user is browsing github and not cloning the repo